docs/develop/sso-in-office-add-ins.md

```diff
@@ -1,69 +1,86 @@
+Users sign in to Office (online, mobile, and desktop platforms) using either their personal Microsoft account or their Microsoft 365 Education or work account. You can take advantage of this and use single sign-on (SSO) to authenticate and authorize the user to your add-in without requiring the user to sign in a second time.
 
 ![An image showing the sign-in process for an add-in.](../images/sso-for-office-addins.png)
 
+## How SSO works at runtime
+
+The following diagram shows how the SSO process works. The blue elements represent Office or the Microsoft identity platform. The gray elements represent the code you write and include the client-side code (task pane) and the server-side code for your add-in.
+
+![A diagram that shows the SSO process.](../images/sso-overview-diagram.svg)
+
+1. In the add-in, your JavaScript code calls the Office.js API [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken_options_). If the user is already signed in to Office, the Office host will return the access token with the claims of the signed in user.
+2. If the user is not signed in, the Office host application opens a dialog box for the user to sign in. Office redirects to the Microsoft identity platform to complete the sign-in process.
+3. If this is the first time the current user has used your add-in, they are prompted to consent.
+4. The Office host application requests the **access token** from the Microsoft identity platform for the current user.
+5. The Microsoft identity platform returns the access token to Office. Office will cache the token on your behalf so that future calls to **getAccessToken** simply return the cached token.
+6. The Office host application returns the **access token** to the add-in as part of the result object returned by the `getAccessToken` call.
+7. The token is both an **access token** and an **identity token**. You can use it as an identity token to parse and examine claims about the user, such as the user's name and email address.
+8. Optionally, the add-in can use the token as an **access token** to make authenticated HTTPS requests to APIs on the server-side. Because the access token contains identity claims, the server can store information associated with the user's identity; such as the user's preferences.
+
 ## Requirements and Best Practices
 
+### Don't cache the access token
+
+Never cache or store the access token in your cient-side code. Always call [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken_options_) when you need an access token. Office will cache the access token (or request a new one if it expired.) This will help to avoid accidentally leaking the token from your add-in.
```

Comment:
```suggestion
Never cache or store the access token in your cient-side code. Always call [getAccessToken](/javascript/api/office-runtime/officeruntime.auth#getAccessToken_options_) when you need an access token. Office caches the access token (or request a new one if it expired.) This helps to avoid accidentally leaking the token from your add-in.
```